crates/router/src/connector/adyenplatform/transformers.rs

```diff
@@ -27,3 +29,19 @@ impl TryFrom<&types::ConnectorAuthType> for AdyenplatformAuthType {
         }
     }
 }
+
+#[derive(Debug, Serialize)]
+pub struct AdyenRouterData<T> {
```

Comment:
we already have adyenRouterData, can we borrow that else make it `AdyenPlatformRouterData`